src/graphql/query/algoliaLoanStatus.graphql

```diff
@@ -9,6 +9,7 @@ query algoliaLoanStatus(
 				id
 				status
 				matchingText
+				minNoteSize
```

Comment:
I'm _ok_ with adding minNoteSize to all these queries, even though some of them are not using it.

Reply:
Good to hear, thank you. I'm admittedly not sure which ones really need it, particularly this one.

Reply:
If there are queries you're sure don't need this addition, I'll gladly remove them.